File: front/mobile/Android/PasswordStorage/mobile/src/androidTest/java/com/ndu/passwordstorage/suite/InstrumentedTestSuite.java
package com.ndu.passwordstorage.suite;

import com.ndu.passwordstorage.ExampleInstrumentedTest;
import com.ndu.passwordstorage.screen.DisplayListActivityTest;

import org.junit.runner.RunWith;
import org.junit.runners.Suite;

@RunWith(Suite.class)
@Suite.SuiteClasses({DisplayListActivityTest.class, ExampleInstrumentedTest.class})
public class InstrumentedTestSuite {
}

File: AndroidApplication/app/src/androidTest/java/com/ndu/passwordstorage/InstrumentedTestSuite.kt
package com.ndu.passwordstorage

import com.ndu.passwordstorage.infrastructure.screen.activity.MainActivityTest
import org.junit.runner.RunWith
import org.junit.runners.Suite

@RunWith(Suite::class)
@Suite.SuiteClasses(
    MainActivityTest::class,
    ExampleInstrumentedTest::class
)
class InstrumentedTestSuite {
}